packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -664,3 +723,21 @@ export enum TagMutability {
   IMMUTABLE = 'IMMUTABLE',
 
 }
+
+/**
+ * Indicates whether server-side encryption is enabled for the object, and whether that encryption is
+ * from the AWS Key Management Service (AWS KMS) or from Amazon S3 managed encryption (SSE-S3).
+ * @see https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html#SysMetadata
+ */
+export enum RepositoryEncryption {
```

Comment:
Can you turn this into an "enum-like" class? https://github.com/aws/aws-cdk/blob/master/docs/DESIGN_GUIDELINES.md#enums 

Or, remove the the error in line 662, since that case is impossible.